src/components/views/rooms/VoiceRecordComposerTile.tsx

```diff
@@ -217,7 +217,7 @@ export default class VoiceRecordComposerTile extends React.PureComponent<IProps,
     public render(): ReactNode {
         if (!this.state.recordingPhase) return null;
 
-        let stopOrRecordBtn;
+        let stopBtn;
         let deleteButton;
         if (!this.state.recordingPhase || this.state.recordingPhase === RecordingState.Started) {
```

Comment:
this condition is a bit pointless, and adds classes which aren't needed

Reply:
I can't get at the lines to offer a github suggestion, but the diff I'm proposing is:

```diff
diff --git a/src/components/views/rooms/VoiceRecordComposerTile.tsx b/src/components/views/rooms/VoiceRecordComposerTile.tsx
index e172fb7f0d..9ab089efe5 100644
--- a/src/components/views/rooms/VoiceRecordComposerTile.tsx
+++ b/src/components/views/rooms/VoiceRecordComposerTile.tsx
@@ -219,20 +219,14 @@ export default class VoiceRecordComposerTile extends React.PureComponent<IProps,

         let stopBtn;
         let deleteButton;
-        if (!this.state.recordingPhase || this.state.recordingPhase === RecordingState.Started) {
-            const classes = classNames({
-                'mx_MessageComposer_button': !this.state.recorder,
-                'mx_MessageComposer_voiceMessage': !this.state.recorder,
-                'mx_VoiceRecordComposerTile_stop': this.state.recorder?.isRecording,
-            });
-
+        if (this.state.recordingPhase === RecordingState.Started) {
             let tooltip = _t("Send voice message");
             if (!!this.state.recorder) {
                 tooltip = _t("Stop recording");
             }

             stopBtn = <AccessibleTooltipButton
-                className={classes}
+                className="mx_VoiceRecordComposerTile_stop"
                 onClick={this.onRecordStartEndClick}
                 title={tooltip}
             />;
```